### PR TITLE
Feature/allowing to move graph back

### PR DESCRIPTION
**Pull request by Augustyniak**

### CODE REVIEW

**Comment by Augustyniak** [comment]
@bartekchlebek Please take a look.

**Comment by Augustyniak** [comment]
🎁

File: EZAudio/EZAudioUtilities.m [hunk 1]
```diff
@@ -670,6 +670,27 @@ + (void)appendBufferRMS:(float *)buffer
 
 //------------------------------------------------------------------------------
 
++ (void)removeEndWithSize:(UInt32)bufferSizeToRemove fromHistoryInfo:(EZPlotHistoryInfo *)historyInfo
+{
+    int32_t bufferSizeToRemoveInBytes = bufferSizeToRemove * sizeof(float);
+
+    int32_t bytesAvailableForRead = 0;
+    float *historyBuffer = TPCircularBufferTail(&historyInfo->circularBuffer, &bytesAvailableForRead); //pointer do czytania
```
Comment (bubski) [inline EZAudio/EZAudioUtilities.m]: `//pointer do czytania` 😂

File: EZAudio/EZAudioUtilities.m [hunk 2]
```diff
@@ -741,4 +760,16 @@ + (EZPlotHistoryInfo *)historyInfoWithDefaultLength:(int)defaultLength
 
 //------------------------------------------------------------------------------
 
++ (UInt32)maximumBufferSizeForAudioUnit:(AudioUnit)audioUnit
+{
+    UInt32 maximumBufferSize;
+    UInt32 propSize = sizeof(maximumBufferSize);
+    [EZAudioUtilities checkResult:AudioUnitGetProperty(audioUnit,
+                                                       kAudioUnitProperty_MaximumFramesPerSlice,                                                       kAudioUnitScope_Global,
```
Comment (bubski) [inline EZAudio/EZAudioUtilities.m]: alignment

**Comment by bubski** [comment]
@Augustyniak 🎾

**Comment by Augustyniak** [comment]
@bartekchlebek Fixed. 🎾

**Pull request merged**